packages/flutter_tools/pubspec.yaml

```diff
@@ -78,7 +78,7 @@ dependencies:
   csslib: 1.0.0 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
   dap: 1.3.0 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
   dds_service_extensions: 2.0.0 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
-  devtools_shared: 10.0.2 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
+  devtools_shared: 10.1.0 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
```

Comment:
@bkonyi looks like this is breaking dds (and thus the tool):

```
/home/chrome-bot/.pub-cache/hosted/pub.dev/dds-4.2.6/lib/src/devtools/handler.dart:166:12: Error: The argument type '({String? secret, String? uri})?' can't be assigned to the parameter type 'DtdInfo?'.
 - 'DtdInfo' is from 'package:devtools_shared/src/common.dart' ('/home/chrome-bot/.pub-cache/hosted/pub.dev/devtools_shared-10.1.0/lib/src/common.dart').
      dtd: dtd,
           ^
```

Reply:
devtools_shared 10.1.0 has been retracted. If you re-run the update this should go back to 10.0.2

Reply:
thanks!